src/main/java/org/sonar/squidbridge/rules/SqaleXmlLoader.java

```diff
@@ -31,14 +37,11 @@
 import org.sonar.api.server.rule.RulesDefinition.NewRepository;
 import org.sonar.api.server.rule.RulesDefinition.NewRule;
 
-import javax.xml.stream.XMLInputFactory;
-import javax.xml.stream.XMLStreamException;
-
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.net.URL;
-import java.util.Map;
-
+/**
+ * @deprecated since 2.7. Will be removed without alternative.
+ * Since SQ 5.6 LTS, SQALE is not used in our model
+ */
+@Deprecated
 @Beta
 public class SqaleXmlLoader {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Do not forget to remove this deprecated code someday. ([squid:S1133](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sslr-squid-bridge%3Asslr-squid-bridge&branch=SSLRSQBR-71&issues=AWHXxlqOaf12pdu-6jFD&open=AWHXxlqOaf12pdu-6jFD)

[ISSUE_KEY]: AWHXxlqOaf12pdu-6jFD